### ISSUE TITLE
Ordered list (Windows Pandoc 1.19.2.1) fails with "missing \item"

### ISSUE THREAD

**Issue by jonathandueck** [jgm/pandoc#5571]
Using Pandoc for Windows (pandoc --version reports: pandoc.exe 1.19.2.1), if I attempt to convert a Markdown file containing an ordered list to PDF via LaTeX, the conversion fails when it reaches the list,. See the following, for example:

`$ pandoc --template=default.latex --latex-engine='C:\cygwin64\usr\local\bin\pdflatex.exe' -o roody.pdf -i roody.md --filter pandoc-citeproc -H preamble.tex
! LaTeX Error: Something's wrong--perhaps a missing \item.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.161   H

pandoc.exe: Error producing PDF
`
This looks like issue #767 (https://github.com/jgm/pandoc/issues/767). However, downloading the current default.latex template and specifying it on the command line, as you can see I've done above, does not solve the issue.

**Comment by jgm** [comment]
Why don't you try this with an up-to-date version of pandoc (we're on 2.2.7), and we provide Windows binaries.  If the problem can be reproduced with the latest version, then we're interested; otherwise, this should be closed.

**Comment by jgm** [comment]
Note that the problem in #767 was precisely that someone was using a template designed for a different version of pandoc than they were running.  That's wht you're doing if you're using the current default.latex template.  To get the template appropriate to your version of pandoc, use `pandoc -D latex > default.latex`.

**Comment by jonathandueck** [comment]
Thanks again for this reply, and apologies for the flak. I had installed the latest version, but it appears that the installer didn't (and I didn't) delete the previous versions. So though I installed the newest 2.7.3 version, my Pandoc command-line was still running the 1.19.2.1 version. I found and deleted the old instances of the command, and the command now works. Apologies for leading us astray here!
